### PR TITLE
test: add test for empty table (refs SFKUI-6500)

### PR DESCRIPTION
**Pull request by NicklasWei**
Buggfix och tester för tom tabell.

### CODE REVIEW

File: packages/vue-labs/src/components/FTable/use-tabstop.ts [hunk 1]
```diff
@@ -156,7 +156,7 @@ export function useTabstop(
         assertRef(tableRef);
         const target = getCellTarget(tableRef.value, 1, 0);
 
-        if (metaRows.value.length === 0) {
+        if (renderOptions.value.focus && metaRows.value.length === 0) {
```
Comment (dvitamin) [inline packages/vue-labs/src/components/FTable/use-tabstop.ts]: Det borde inte behövas någon if-else här, enbart raden

```
activateCell(target, { focus: renderOptions.value.focus });
```
